[2024-01-21T11:36:02.708-0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_sql_pipeline.create_table manual__2024-01-21T19:35:59.630505+00:00 [queued]>
[2024-01-21T11:36:02.716-0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_sql_pipeline.create_table manual__2024-01-21T19:35:59.630505+00:00 [queued]>
[2024-01-21T11:36:02.716-0800] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-21T11:36:02.716-0800] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-01-21T11:36:02.717-0800] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-21T11:36:02.741-0800] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): create_table> on 2024-01-21 19:35:59.630505+00:00
[2024-01-21T11:36:02.743-0800] {standard_task_runner.py:55} INFO - Started process 30730 to run task
[2024-01-21T11:36:02.747-0800] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'executing_sql_pipeline', 'create_table', 'manual__2024-01-21T19:35:59.630505+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/executing_sql_pipeline.py', '--cfg-path', '/tmp/tmp5jf4x8nt']
[2024-01-21T11:36:02.748-0800] {standard_task_runner.py:83} INFO - Job 89: Subtask create_table
[2024-01-21T11:36:02.818-0800] {task_command.py:389} INFO - Running <TaskInstance: executing_sql_pipeline.create_table manual__2024-01-21T19:35:59.630505+00:00 [running]> on host Mittu.
[2024-01-21T11:36:02.941-0800] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=madhu
AIRFLOW_CTX_DAG_ID=executing_sql_pipeline
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-21T19:35:59.630505+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-21T19:35:59.630505+00:00
[2024-01-21T11:36:02.945-0800] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
                    id INTEGER PRIMARY KEY,
                    name VARCHAR(50) NOT NULL,
                    age INTEGER NOT NULL,
                    city VARCHAR (50),
                    is_active BOOLEAN DEFAULT true,
                    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
            );
        
[2024-01-21T11:36:02.959-0800] {base.py:73} INFO - Using connection ID 'my_sqlite_conn' for task execution.
[2024-01-21T11:36:03.059-0800] {base.py:73} INFO - Using connection ID 'my_sqlite_conn' for task execution.
[2024-01-21T11:36:03.060-0800] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                    id INTEGER PRIMARY KEY,
                    name VARCHAR(50) NOT NULL,
                    age INTEGER NOT NULL,
                    city VARCHAR (50),
                    is_active BOOLEAN DEFAULT true,
                    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
            );
        , parameters: None
[2024-01-21T11:36:03.065-0800] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=executing_sql_pipeline, task_id=create_table, execution_date=20240121T193559, start_date=20240121T193602, end_date=20240121T193603
[2024-01-21T11:36:03.121-0800] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-21T11:36:03.145-0800] {taskinstance.py:2596} INFO - 2 downstream tasks scheduled from follow-on schedule check
